packages/fxa-payments-server/src/components/PaymentLegalBlurb/index.tsx

```diff
@@ -34,7 +34,9 @@ const PaypalPaymentLegalBlurb = () => (
             href="https://www.paypal.com/webapps/mpp/ua/privacy-full"
             target="_blank"
             rel="noopener noreferrer"
```

Comment:
I know this is unrelated to your PR but we have [`LinkExternal`](https://github.com/mozilla/fxa/blob/main/packages/fxa-react/components/LinkExternal/index.tsx) for a11y friendly, safe external links. We might need a follow up to audit our use of external links across packages for consistency!

Reply:
Cool, created a Jira for this with related issue #12468